concourse/scripts/concourse-lite-environment.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+PROJECT_DIR=$(cd "${SCRIPT_DIR}/../.." && pwd)
+
+hashed_password() {
+  echo "$1" | shasum -a 256 | base64 | head -c 32
+}
+
+DEPLOY_ENV=${1:-${DEPLOY_ENV:-}}
+if [ -z "${DEPLOY_ENV}" ]; then
+  echo "Must specify DEPLOY_ENV as \$1 or environment variable" 1>&2
+  exit 1
+fi
+
+AWS_ACCOUNT=${AWS_ACCOUNT:-dev}
+
+CONCOURSE_URL="${CONCOURSE_URL:-http://localhost:8080}"
+FLY_TARGET="${FLY_TARGET:-${DEPLOY_ENV}-bootstrap}"
+FLY_CMD="${PROJECT_DIR}/bin/fly-bootstrap"
+
+CONCOURSE_ATC_USER=${CONCOURSE_ATC_USER:-admin}
+if [ -z "${CONCOURSE_LITE_ATC_PASSWORD:-}" ]; then
+    CONCOURSE_LITE_ATC_PASSWORD=$(hashed_password "$(gpg --export-secret-key -a)")
```

Comment:
Some of us talked about this at the desks. There's discomfort about the Security Group being the only restriction to accessing the bootstrap Concourse. We could also change the bind address for ATC so that it's only accessible over loopback (SSH tunnel).

However @Jonty suggested that we could use something from the AWS API that's not easily determinable by people outside the team - such as account ID, user ARN, or user ID. @combor and @saliceti are looking into it.